src/bp/common/action.ts

```diff
@@ -30,3 +31,15 @@ export const parseActionInstruction = (actionInstruction: string): ActionInstruc
     actionServerId
   }
 }
+
+export const extractEventCommonArgs = (event: IO.IncomingEvent, args?: any) => {
+  return {
+    ...(args ?? {}),
+    event,
+    user: event.state.user ?? {},
+    session: event.state.session ?? {},
+    temp: event.state.temp ?? {},
+    bot: event.state.bot ?? {},
+    workflow: event.state.workflow ?? {}
+  }
```

Comment:
the "workflow" parameter was missing in one place, so instead of just adding it there, i've changed it so we do not make that error in the future (since there's a couple of places where we build those "shortcuts"